cob_cartesian_controller/include/cob_cartesian_controller/cartesian_controller.h

```diff
@@ -48,8 +60,6 @@ class CartesianController
 	
 	// Main functions
 	void pose_path_broadcaster(std::vector <geometry_msgs::Pose> *poseVector);
-	void linear_interpolation(std::vector <geometry_msgs::Pose> *poseVector,geometry_msgs::Pose, geometry_msgs::Pose,double,double,std::string,bool); 
-	void circular_interpolation(std::vector<geometry_msgs::Pose> *poseVector,double,double,double,double,double,double,double,double,double,double,double,std::string);
 	void move_ptp(geometry_msgs::Pose targetPose, double epsilon);
 	void hold_position(geometry_msgs::Pose);
 	
```

Comment:
Helper functions should go into a new `helper_functions.h`.
They can be global functions and namespaced `cartesian_controller`